### ISSUE TITLE
MOOSE autocompletion not working in VScode

### ISSUE THREAD

**Issue by YKumar1999** [idaholab/moose-language-support#21]
Hi,
I am opened BISON input file in VSCode. Somehow, autocompletion option is not fully working for me, when I tried CNTRL + Space option, its not working. While typing the command line, it works but CNTRL + space option is not working
<img width="609" alt="Screenshot 2024-01-25 at 9 31 49 AM" src="https://github.com/idaholab/moose-language-support/assets/143427553/00939dfd-56d9-4c0c-a064-86f36f1aef50">

**Comment by nihonjin2024** [comment]
I'm having the same problem.

**Comment by dschwen** [comment]
Yeah, I ran into this as well. This is not an issue with the extension. You most likely have a system hotkey enabled for CTRL+Space that captures the keypress and doesn't let it go through to VSCode. I had to uncheck the checkmark next to "Select the previous input source".

![Image](https://github.com/user-attachments/assets/2e065543-8835-41ca-8b74-84a7f0cebbd0)
